docs/content/preview/yugabyte-platform/create-deployments/read-replicas.md

```diff
@@ -11,42 +11,35 @@ menu:
 
 {{< note title="Note" >}}
 YugabyteDB Anywhere does not support read replica configuration for Kubernetes and OpenShift cloud providers.
 {{< /note >}}
 
-## Create the universe
+## Create a universe with a read replica
 
-You start by navigating to **Dashboard** and clicking **Create Universe**. Use the **Primary Cluster > Cloud Configuration** page to enter the following values to create a primary cluster on [GCP](../../configure-yugabyte-platform/set-up-cloud-provider/gcp/) provider:
+You start by navigating to **Dashboard** and clicking **Create Universe**. Use the **Primary Cluster** tab to enter the following values to create a primary cluster on [GCP](../../configure-yugabyte-platform/set-up-cloud-provider/gcp/) provider:
 
 - Enter a universe name as helloworld3.
 - Enter the set of regions as Oregon.
 - Set the replication factor to 3.
 - Set instance type to n1-standard-8
-- Add the configuration flag for YB-Master and YB-TServer as `leader_failure_max_missed_heartbeat_periods` 10. Since the data is globally replicated, remote procedure call (RPC) latencies are higher. You can use this flag to increase the failure detection interval in such a high-RPC latency deployment.<br>
-
-  ![Create Primary Cluster on GCP](/images/ee/primary-cluster-creation.png)
+- Add the configuration flag for YB-Master and YB-TServer as `leader_failure_max_missed_heartbeat_periods` 10. As the data is globally replicated, remote procedure call (RPC) latencies are higher. You can use this flag to increase the failure detection interval in such a high-RPC latency deployment.
 
-The next step is to click **Configure Read Replica** and then specify the following on the **Read Replica > Cloud Configuration** page to create a read replica cluster on [AWS](../../configure-yugabyte-platform/set-up-cloud-provider/aws/):
+The next step is to click **Configure Read Replica** and then specify the following on the **Read Replica** tab to create a read replica cluster on [AWS](../../configure-yugabyte-platform/set-up-cloud-provider/aws/):
 
 - Enter the set of regions as US East.
-- Set the replication factor to 3.
+- Set the number of read replicas to 3.
 - Set the instance type to c4.large.
 
-Since you do not need to a establish a quorum for read replica clusters, the replication factor can be
-either even or odd.
-
 To finish the process, click **Create**.
 
-## Examine the universe
-
-While the universe is being created, **Dashboard** should look similar to the following illustration:
-
-![Universe Waiting to Create](/images/ee/universe-waiting.png)
-
-Once the universe has been created, **Dashboard** displays the primary and read replica cluster information, as well as shows the distinct clusters on the map.
+After the universe has been created, **Dashboard** displays the primary and read replica cluster information, as well as shows the distinct clusters on the map.
```

Comment:
Dashboard usually displays only Primary Cluster information, I agree that map display distinct cluster info though

Reply:
Think we can just drop it really